pkg/sql/catalog/systemschema/system.go

```diff
@@ -345,6 +345,17 @@ CREATE TABLE system.sqlliveness (
     expiration       DECIMAL NOT NULL,
   	FAMILY fam0_session_id_expiration (session_id, expiration)
 )`
+
+	LongRunningMigrationsTableSchema = `
+CREATE TABLE system.long_running_migrations (
```

Comment:
Probably not worth doing now, but if we're imagining that this subsystem will eventually grow to envelop pkg/sqlmigrations, maybe a better table name is in order? `internal_migrations`? 

Again, probably not worth introducing new names for, but curious what you think. I think it would also tie in better with the fact that we've named the top-level package as simply "migrations". I don't think we've used "long-running" anywhere in code before this PR.

Reply:
Done, I've removed "long running" in all of its forms. Everything is now just `migrations`.